src/components/tooltip/tooltip.svelte

```diff
@@ -54,6 +58,27 @@
     })
   }
 
+  let contentHovered = false
+
+  const debouncedMouseleave = (event: MouseLeaveEvent) => {
+    let debounce
+
+    return () => {
+      clearTimeout(debounce)
+
+      debounce = setTimeout(() => {
+        if (!contentHovered) {
+          setVisible(false)
+        }
+      }, 300)
+    }
+  }
```

Comment:
Gives the cursor some time to hover over the content in the tooltip, in case there is some action that can be done in the cursor.